pwndbg/commands/procinfo.py

```diff
@@ -241,14 +270,14 @@ def procinfo() -> None:
 
     print("%-10s %s" % ("ppid", proc.ppid))
 
-    if not pwndbg.gdblib.android.is_android():
-        print("%-10s %s" % ("uid", proc.uid))
-        print("%-10s %s" % ("gid", proc.gid))
-        print("%-10s %s" % ("groups", proc.groups))
-    else:
-        print("%-10s %s" % ("uid", list(map(pwndbg.lib.android.aid_name, proc.uid))))
-        print("%-10s %s" % ("gid", list(map(pwndbg.lib.android.aid_name, proc.gid))))
-        print("%-10s %s" % ("groups", list(map(pwndbg.lib.android.aid_name, proc.groups))))
+    # if pwndbg.gdblib.android.is_android():
+    print("%-10s %s" % ("uid", proc.uid))
+    print("%-10s %s" % ("gid", proc.gid))
+    print("%-10s %s" % ("groups", proc.groups))
+    # else:
+    #     print("%-10s %s" % ("uid", list(map(pwndbg.lib.android.aid_name, proc.uid))))
+    #     print("%-10s %s" % ("gid", list(map(pwndbg.lib.android.aid_name, proc.gid))))
+    #     print("%-10s %s" % ("groups", list(map(pwndbg.lib.android.aid_name, proc.groups))))
```

Comment:
Why commented out?

Reply:
Tbh I don't know if this code worked on Android. Maybe we should just remove it. Ideally, this shoudl be in another PR :(